csv2geojson.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import csv
+import json
+
+argv = sys.argv[1:]
+if len(argv) == 0:
+    print("no csv file specified")
+    sys.exit(2)
+csvfile = argv[0]
+
+features = []
+with open(csvfile, 'r', encoding='utf-8') as file:
+    reader = csv.DictReader(file)
+    for row in reader:
+        if 'uid' not in row or row['uid'] == '' :
+            print(f'Attribute \'uid\' missing in row {row}')
+            continue
+        if 'lat' not in row or 'lon' not in row:
+            print(f'Attributes \'lat\'/\'lon\' missing in row {row}')
+            continue
+        properties = { 'uid': row['uid'] }
+        for column in ['address', 'type']:
+            if column in row and row[column] != '':
+                properties[column] = row[column]
+        for column in ['max_height', 'max_width', 'max_depth']:
+            if column in row and row[column] != '' and row[column].isdigit():
+                properties[column] = int(row[column])
+        if 'park_and_ride_type' in row and row['park_and_ride_type'] != '':
```

Comment:
You can just write `if not park_and_ride_type.get('park_and_ride_type')` to simplify this (same at other places)